test/e2e/common/runtime.go

```diff
@@ -259,37 +260,37 @@ while true; do sleep 1; done
 				waiting     bool
 			}{
 				{
-					description: "should not be able to pull image from invalid registry",
+					description: "should not be able to pull image from invalid registry [LinuxOnly]",
```

Comment:
Why is this LinuxOnly? The image says alpine, but doesn't actually exist

Reply:
Same question for all the following test cases also

Reply:
Good point. this passes.